schema/theme.schema.json

```diff
@@ -58,28 +59,29 @@
         "heading-color-inverted": {
           "type": "string",
           "title": "Heading colour - inverted",
-          "description": "Global title colour inverted. Either a lightened/darkened alternative to Title colour",
+          "description": "Global title colour inverted. Either a lightened / darkened alternative to Title colour.",
           "default": "",
           "_backboneForms": "ColourPicker"
         }
       }
     },
+
     "_items": {
       "type": "object",
-      "title": "Items (components)",
+      "title": "Content Items (components)",
       "default": {},
       "properties": {
         "item-color": {
           "type": "string",
           "title": "Item colour",
-          "description": "Defines the clickable areas or points of interest in components",
+          "description": "Defines the clickable areas or points of interest in components.",
```

Comment:
Minor but 'selectable' is more inclusive word choice than 'clickable'.

Reply:
This should now be resolved